CMakeLists.txt

```diff
@@ -157,7 +157,7 @@ endif()
 
 # Python
 include(VolkPython) #sets PYTHON_EXECUTABLE and PYTHON_DASH_B
-VOLK_PYTHON_CHECK_MODULE("python >= 3.4" sys "sys.version.split()[0] >= '3.4'" PYTHON_MIN_VER_FOUND)
+VOLK_PYTHON_CHECK_MODULE("python >= 3.4" sys "sys.version_info > (3, 4)" PYTHON_MIN_VER_FOUND)
```

Comment:
To be fair, should be ">=" since in theory we could use Python 3.4.0 & this change would prevent that version from being accepted.